<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>551</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>定时器</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="4">
     <widget class="QLCDNumber" name="lcdSecond"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>定时周期</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="bntSetC">
      <property name="text">
       <string>设置周期</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="bntStart">
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSpinBox" name="spinBoxSetT">
      <property name="suffix">
       <string> ms</string>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLCDNumber" name="lcdHour"/>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>  流逝时间</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="bntStop">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLCDNumber" name="lcdMiniute"/>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>411</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>日历选择</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>选择的日期:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineShowTime"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QCalendarWidget" name="calendarWidget"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="bntCancel">
      <property name="text">
       <string>退  出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>551</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>日期时间</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="bntReadTime">
      <property name="text">
       <string>读取当前日期</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>字符串显示</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>时 间</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineTime"/>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="bntSetTime">
      <property name="text">
       <string>设置时间</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>日 期</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineDate"/>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="bntSetDate">
      <property name="text">
       <string>设置日期</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>日期时间</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineDT"/>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="bntSetDT">
      <property name="text">
       <string>设置日期时间</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTimeEdit" name="timeEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
